selfdrive/ui/layouts/settings/settings.py

```diff
@@ -43,7 +45,7 @@ class PanelType(IntEnum):
 @dataclass
 class PanelInfo:
   name: str
-  instance: object
+  instance: Widget
```

Comment:
I just did this to fix a type issue in my IDE

Reply:
It was the Pylance VSCode extension, if you're wondering. It shows quite a few type issues on a lot of the files, but I can't fix them all.